hashNumBuckets: 2000000
hashBucketSize: 10
hashMaxCollisionLinkedListSize: 100
numSDFBlocks: 200000
SDFBlockSize: 8
virtualVoxelSize: 0.08
numOccupiedBlocks: 0
maxIntegrationDistance: 20
truncScale: 5
truncation: 5
integrationWeightSample: 5
integrationWeightMax: 5

marchingCubesMaxNumTriangles: 3000000
SDFMarchingCubeThreshFactor: 0